src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler.properties

```diff
@@ -1861,7 +1861,7 @@ compiler.warn.finally.cannot.complete=\
 
 # 0: name
 compiler.warn.poor.choice.for.module.name=\
-    module name component {0} should avoid terminal digits
+    module name component {0} ends with a digit. Module names should not encode version information in module names
```

Comment:
Duplicate text

_Module names_. should not ... in _module names_

Reply:
fixed in eed69a0